ingress/controllers/nginx/examples/default/rc-default.yaml

```diff
@@ -14,11 +14,18 @@ spec:
         k8s-app: nginx-ingress-lb
         name: nginx-ingress-lb
     spec:
-      terminationGracePeriodSeconds: 60      
+      terminationGracePeriodSeconds: 60
       containers:
       - image: gcr.io/google_containers/nginx-ingress-controller:0.8.2
         name: nginx-ingress-lb
         imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 80
```

Comment:
I've removed the readiness probe changes for now, we can handle it in a follow up pr and discuss convention on a bug